packages/server/src/fhircast/websocket.test.ts

```diff
@@ -166,16 +167,21 @@ describe('FHIRcast WebSocket', () => {
             expect(obj['hub.topic']).toBe(topic);
           })
           .expectJson((obj) => {
-            // Event message
-            expect(obj.event['hub.topic']).toBe(topic);
-            expect(obj.event['hub.event']).toBe('heartbeat');
+            expect(obj).toMatchObject({
+              id: expect.any(String),
+              timestamp: expect.any(String),
+              event: {
+                context: [{ key: 'period', decimal: '10' }],
+                'hub.event': 'heartbeat',
+              },
+            });
           })
           .sendJson({ ok: true })
           .close()
           .expectClosed();
       }));
 
-    test('Check that timer and promises are cleaned up after no topics active', () =>
+    test('Make sure that we only get one heartbeat per tick for a given topic', () =>
```

Comment:
I repurposed this test since I realized it was the exact same test as the one above with a different name 😬 So gave it a little more purpose now